src/components/BlogCard/index.js

```diff
@@ -21,56 +36,100 @@ const stripMarkdown = (markdown) => {
   return text;
 };
 
-export const BlogCard = ({ node, ...rest }) => (
-  <Box
-    margin={{
-      left: 'large',
-      right: 'large',
-    }}
-    pad={{
-      bottom: 'large',
-      top: 'large',
-    }}
-    gap="medium"
-    border={{
-      side: 'bottom',
-      color: 'light-2',
-      size: 'medium',
+const columns = {
+  small: ['auto'],
+  medium: ['auto', 'auto'],
+  large: ['auto', 'auto', 'auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto', 'auto'],
+};
+
+const rows = {
+  small: ['auto', 'auto', 'auto'],
+  medium: ['auto', 'auto'],
+  large: ['auto'],
+  xlarge: ['auto'],
+};
+
+export const AllBlogsCard = ({
+  children,
+  overrideColumns,
+  overrideRows,
```

Comment:
doesn't look like you actually use these and you should simply call them rows/columns and deal with it below. Keep the parameters clean.

Reply:
I cleaned the parameters